### PR TITLE
feat(mastodon): Add env for Mastodon

### PR DESCRIPTION
**Pull request by MorpheusH3x**
Make all necessary changes to add a Mastodon entry

### CODE REVIEW

**Comment by Ornias1993** [comment]
Nope, Github best practice is to upstream your changes here first:
https://github.com/sethcottle/littlelink

**Comment by MorpheusH3x** [comment]
Ok, so I'll make my changes there.
All the change made in the littlelink repo are reflected in the littlelink-server one?

**Comment by Ornias1993** [comment]
> Ok, so I'll make my changes there.
> All the change made in the littlelink repo are reflected in the littlelink-server one?

No it would still require a maintainer to port the changes afterwards, unless the maintainer has setup automation for it.

**Comment by MorpheusH3x** [comment]
So I will make the same modification on the repo of littlelink and keep this PR open to kind of "automatically" port the changes from the littlelink repo to the littlelink-server one ?

File: docker-compose.yml [hunk 1]
```diff
@@ -19,6 +19,7 @@ services:
       - BIO=Hey! Just a place where you can connect with me!
       - GITHUB=https://github.com/timothystewart6
       - TWITTER=https://twitter.com/TechnoTimLive
+      - MASTODON=https://mastodon.social/@Gargron
```
Comment (timothystewart6) [inline docker-compose.yml:22]: Can you remove the example?  Keeping the example as is for now

**Comment by timothystewart6** [comment]
@MorpheusH3x   Thank you!  One small comment but after that good to go.

File: docker-compose.yml (hunk 1 above)
Comment (MorpheusH3x) [reply docker-compose.yml]: I made the mastodon as the twitter one: `MASTODON=https://mastodon.social/TechnoTimLive`.
Hope I did not miss understand you.

File: docker-compose.yml [hunk 2]
```diff
@@ -19,7 +19,7 @@ services:
       - BIO=Hey! Just a place where you can connect with me!
       - GITHUB=https://github.com/timothystewart6
       - TWITTER=https://twitter.com/TechnoTimLive
-      - MASTODON=https://mastodon.social/@Gargron
+      - MASTODON=https://mastodon.social/TechnoTimLive
```
Comment (timothystewart6) [inline docker-compose.yml:22]: any way you can just remove the line?

Comment (MorpheusH3x) [reply docker-compose.yml:22]: Why removing the line ? 
It is needed to easily modify the target link in the docker-compose, no ?

Comment (timothystewart6) [reply docker-compose.yml:22]: This is only the example.  All good.  I am going to rework examples soon.  Thank you!

Comment (MorpheusH3x) [reply docker-compose.yml:22]: I just can not figure out your mastodon account so I take the maintainer's one.

**Pull request merged**